webviz_subsurface/_abbreviations/reservoir_simulation.py

```diff
@@ -71,3 +73,41 @@ def simulation_vector_description(vector: str):
         )
 
     return description
+
+
+def historical_vector(
+    vector: str,
+    smry_meta: Optional[pd.DataFrame] = None,
```

Comment:
Maybe add some docstring explanation of the `smry_meta` input, and how that is used in this function.